server/models/Notification.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const { Schema, ObjectId } = mongoose;
+
+const notificationSchema = new Schema({
+  recipient: { type: ObjectId, required: true },
+  code: Number,
+  title: String,  // "you have a new request from {user name}!"
+  date: { type: Date, default: Date.now },  // epoch time integer
+  link: String, // request or review? requests/:reviewId or reviews/:reviewId --> HOW DO I GIVE THIS A DEFAULT VALUE BASED ON ITS ID?
```

Comment:
there is probably no default value for this, if it doesn't have a value, the front-end can just not click to any part of the app :)